packages/styles/components/_c.button.scss

```diff
@@ -39,6 +39,10 @@
   }
 
   &--square {
+    @include set-from-screen("l") {
+      padding: 0;
+    }
```

Comment:
As there is already a `padding: 0` inside this selector, so this one is not necessary I think.

Reply:
I added this because the mixin `set-button-sizes` override the padding. 
I had to add the `padding: 0 ` to make sure the button square stay as it is